packages/datadog_flutter_plugin/android/src/main/kotlin/com/datadoghq/flutter/DatadogSdkPlugin.kt

```diff
@@ -73,7 +74,9 @@ class DatadogSdkPlugin : FlutterPlugin, MethodCallHandler {
     val rumPlugin: DatadogRumPlugin = DatadogRumPlugin()
 
     override fun onAttachedToEngine(
-        @NonNull flutterPluginBinding: FlutterPlugin.FlutterPluginBinding
+        @SuppressLint("KotlinNullnessAnnotation")
+        @NonNull
```

Comment:
Why this annotation is needed if nullness already specified by type?

Reply:
It's in base type and if I take it out it warns that I'm not overriding the base properly?

Reply:
It's a good call, let me see what I can do.